src/containers/SellerProductsContainer.jsx

```diff
@@ -167,41 +167,45 @@ const SellerProductsContainer = () => {
             )}
 
             {/* Product in my collection */}
-           <div className='flex flex-wrap justify-evenly mt-5'>
-             {myCollection?.map((product, index) => {
-              return (
-                <div
-                  key={product.id}
-                  className='sellingProd__card flex  mb-5 '
-                  onClick={() => {
-                    return getProduct(index);
-                  }}
-                >
-                  <div className='sellingProd__cardImage flex '>
-                    <img src={product.image[0]} alt='Product' />
-                  </div>
-                  <div className='sellingProd__cardDetails flex grow justify-between flex-col'>
-                    <div className='sellingProd__cardDetailsPara'>
-                      <div>
-                        <p className='flex justify-start items-center pl-[10px] py-4'>
-                          <span> {product.name}</span>
-                        </p>
-                      </div>
-                      <div>
-                        <p className='flex justify-start items-center pl-[10px] py-4'>
-                          $<span> {product.price}</span>
-                        </p>
+            <div className='flex flex-wrap justify-evenly mt-5'>
+              {myCollection
+                ?.filter((product) => {
```

Comment:
It is really great that you filter out products that are out of stock before displaying them on the UI.